Error in initialize():  NoClassDefFoundError: org.apache.commons.lang3.StringUtils

Test with hbase toolkit v3.9.0 failed.

```
python3 -u -m unittest test_hbase.TestDistributedDownloadToolkit
<class 'test_hbase.TestDistributedDownloadToolkit'>
Download: https://github.com//IBMStreams/streamsx.hbase/releases/download/v3.9.0/streamsx.hbase.toolkits-3.9.0-20200924-1059.tgz
100% [........................................................................] 26267002 / 26267002
  <toolkit name="com.ibm.streamsx.hbase" version="3.9.0">

------ test_hbase_operators ------
Job 3 PE 3 launch count > 1: 4
Job 3 wait for healthy actually failed
Job 3 Wait for healthy failed
F

```

This is caused by `java.lang.NoClassDefFoundError: org.apache.commons.lang3.StringUtils` in initialize() of the HBase operator.

```
Exception in thread "Thread-11" java.lang.NoClassDefFoundError: org.apache.commons.lang3.StringUtils
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.getName(MutableMetricsFactory.java:134)
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.getInfo(MutableMetricsFactory.java:130)
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.newForField(MutableMetricsFactory.java:45)
	at org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.add(MetricsSourceBuilder.java:147)
	at org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.<init>(MetricsSourceBuilder.java:69)
	at org.apache.hadoop.metrics2.lib.MetricsAnnotations.newSourceBuilder(MetricsAnnotations.java:43)
	at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.register(MetricsSystemImpl.java:223)
	at org.apache.hadoop.metrics2.MetricsSystem.register(MetricsSystem.java:71)
	at org.apache.hadoop.security.UserGroupInformation$UgiMetrics.create(UserGroupInformation.java:149)
	at org.apache.hadoop.security.UserGroupInformation.<clinit>(UserGroupInformation.java:265)
	at org.apache.hadoop.conf.Configuration$Resource.getRestrictParserDefault(Configuration.java:284)
	at org.apache.hadoop.conf.Configuration$Resource.<init>(Configuration.java:257)
	at org.apache.hadoop.conf.Configuration$Resource.<init>(Configuration.java:249)
	at org.apache.hadoop.conf.Configuration.addResource(Configuration.java:955)
	at com.ibm.streamsx.hbase.HBASEOperator.getConnection(HBASEOperator.java:413)
	at com.ibm.streamsx.hbase.HBASEOperator.initialize(HBASEOperator.java:407)
	at com.ibm.streamsx.hbase.HBASEOperatorWithInput.initialize(HBASEOperatorWithInput.java:133)
	at com.ibm.streamsx.hbase.HBASEPutDelete.initialize(HBASEPutDelete.java:177)
	at com.ibm.streamsx.hbase.HBASEPut.initialize(HBASEPut.java:224)
	at com.ibm.streams.operator.internal.runtime.api.OperatorAdapter.initialize(OperatorAdapter.java:745)
	at com.ibm.streams.operator.internal.jni.JNIBridge.<init>(JNIBridge.java:277)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.StringUtils
	at java.net.URLClassLoader.findClass(URLClassLoader.java:609)
	at java.lang.ClassLoader.loadClassHelper(ClassLoader.java:926)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:871)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:854)
	... 21 more
Exception in thread "Thread-11" java.lang.NoClassDefFoundError: org.apache.commons.lang3.StringUtils
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.getName(MutableMetricsFactory.java:134)
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.getInfo(MutableMetricsFactory.java:130)
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.newForField(MutableMetricsFactory.java:45)
	at org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.add(MetricsSourceBuilder.java:147)
	at org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.<init>(MetricsSourceBuilder.java:69)
	at org.apache.hadoop.metrics2.lib.MetricsAnnotations.newSourceBuilder(MetricsAnnotations.java:43)
	at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.register(MetricsSystemImpl.java:223)
	at org.apache.hadoop.metrics2.MetricsSystem.register(MetricsSystem.java:71)
	at org.apache.hadoop.security.UserGroupInformation$UgiMetrics.create(UserGroupInformation.java:149)
	at org.apache.hadoop.security.UserGroupInformation.<clinit>(UserGroupInformation.java:265)
	at org.apache.hadoop.conf.Configuration$Resource.getRestrictParserDefault(Configuration.java:284)
	at org.apache.hadoop.conf.Configuration$Resource.<init>(Configuration.java:257)
	at org.apache.hadoop.conf.Configuration$Resource.<init>(Configuration.java:249)
	at org.apache.hadoop.conf.Configuration.addResource(Configuration.java:955)
	at com.ibm.streamsx.hbase.HBASEOperator.getConnection(HBASEOperator.java:413)
	at com.ibm.streamsx.hbase.HBASEOperator.initialize(HBASEOperator.java:407)
	at com.ibm.streamsx.hbase.HBASEOperatorWithInput.initialize(HBASEOperatorWithInput.java:133)
	at com.ibm.streamsx.hbase.HBASEPutDelete.initialize(HBASEPutDelete.java:177)
	at com.ibm.streamsx.hbase.HBASEPut.initialize(HBASEPut.java:224)
	at com.ibm.streams.operator.internal.runtime.api.OperatorAdapter.initialize(OperatorAdapter.java:745)
	at com.ibm.streams.operator.internal.jni.JNIBridge.<init>(JNIBridge.java:277)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.StringUtils
	at java.net.URLClassLoader.findClass(URLClassLoader.java:609)
	at java.lang.ClassLoader.loadClassHelper(ClassLoader.java:926)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:871)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:854)
	... 21 more
Exception in thread "Thread-11" java.lang.NoClassDefFoundError: org.apache.commons.lang3.StringUtils
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.getName(MutableMetricsFactory.java:134)
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.getInfo(MutableMetricsFactory.java:130)
	at org.apache.hadoop.metrics2.lib.MutableMetricsFactory.newForField(MutableMetricsFactory.java:45)
	at org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.add(MetricsSourceBuilder.java:147)
	at org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.<init>(MetricsSourceBuilder.java:69)
	at org.apache.hadoop.metrics2.lib.MetricsAnnotations.newSourceBuilder(MetricsAnnotations.java:43)
	at org.apache.hadoop.metrics2.impl.MetricsSystemImpl.register(MetricsSystemImpl.java:223)
	at org.apache.hadoop.metrics2.MetricsSystem.register(MetricsSystem.java:71)
	at org.apache.hadoop.security.UserGroupInformation$UgiMetrics.create(UserGroupInformation.java:149)
	at org.apache.hadoop.security.UserGroupInformation.<clinit>(UserGroupInformation.java:265)
	at org.apache.hadoop.conf.Configuration$Resource.getRestrictParserDefault(Configuration.java:284)
	at org.apache.hadoop.conf.Configuration$Resource.<init>(Configuration.java:257)
	at org.apache.hadoop.conf.Configuration$Resource.<init>(Configuration.java:249)
	at org.apache.hadoop.conf.Configuration.addResource(Configuration.java:955)
	at com.ibm.streamsx.hbase.HBASEOperator.getConnection(HBASEOperator.java:413)
	at com.ibm.streamsx.hbase.HBASEOperator.initialize(HBASEOperator.java:407)
	at com.ibm.streamsx.hbase.HBASEOperatorWithInput.initialize(HBASEOperatorWithInput.java:133)
	at com.ibm.streamsx.hbase.HBASEPutDelete.initialize(HBASEPutDelete.java:177)
	at com.ibm.streamsx.hbase.HBASEPut.initialize(HBASEPut.java:224)
	at com.ibm.streams.operator.internal.runtime.api.OperatorAdapter.initialize(OperatorAdapter.java:745)
	at com.ibm.streams.operator.internal.jni.JNIBridge.<init>(JNIBridge.java:277)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.StringUtils
	at java.net.URLClassLoader.findClass(URLClassLoader.java:609)
	at java.lang.ClassLoader.loadClassHelper(ClassLoader.java:926)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:871)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:854)
	... 21 more
```

--- a/com.ibm.streamsx.hbase/CHANGELOG.md
+++ b/com.ibm.streamsx.hbase/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changes
-=======
+=========
+
+## v3.9.1:
+* [#136](https://github.com/IBMStreams/streamsx.hbase/issues/136) 3rd party library commons-lang3 added
+
 ## v3.9.0:
 * [#133](https://github.com/IBMStreams/streamsx.hbase/issues/133) The Vulnerability issues for 3rd party libraries: 
 ```

--- a/com.ibm.streamsx.hbase/info.xml
+++ b/com.ibm.streamsx.hbase/info.xml
@@ -223,8 +223,12 @@ The jar library zookeeper-3.4.13.jar has been replaced with **zookeeper-3.4.6.ja
 
 * The Vulnerability issues for 3rd party libraries have been fixed and the pom.xml updated to use hadoop client version 3.3 JAR libraries.
 
+++ What is new in version 3.9.1
+
+* 3rd party library commons-lang3 added
+
 </info:description>
-    <info:version>3.9.0</info:version>
+    <info:version>3.9.1</info:version>
     <info:requiredProductVersion>4.0.0.0</info:requiredProductVersion>
   </info:identity>
   <info:dependencies/>

--- a/com.ibm.streamsx.hbase/pom.xml
+++ b/com.ibm.streamsx.hbase/pom.xml
@@ -11,7 +11,7 @@
 	<groupId>com.ibm.streamsx.hbase</groupId>
 	<artifactId>streamsx.hbase</artifactId>
 	<packaging>jar</packaging>
-	<version>3.9.0</version> 
+	<version>3.9.1</version> 
 	<name>com.ibm.streamsx.hbase</name>
 	<repositories>
 		<repository>
@@ -86,6 +86,17 @@
 			<groupId>commons-lang</groupId>
 			<artifactId>commons-lang</artifactId>
 			<version>2.6</version>
+			<exclusions>
+				<exclusion>
+					<groupId>*</groupId>
+					<artifactId>*</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.11</version>
 			<exclusions>
 				<exclusion>
 					<groupId>*</groupId>

